openml/datasets/functions.py

```diff
@@ -165,6 +165,36 @@ def _get_cache_directory(dataset: OpenMLDataset) -> str:
     return _create_cache_directory_for_id(DATASETS_CACHE_DIR_NAME, dataset.dataset_id)
 
 
+def list_qualities(verbose=False):
+    """ Return list of data qualities available.
+
+    The function performs an API call to retrieve the entire list of
+    data qualities that are available are computed on the datasets uploaded.
+
+    Parameters
+    ----------
+    verbose : bool (default=False)
+        If True, prints out the list with an index
+
+    """
+    api_call = "data/qualities/list"
+    xml_string = openml._api_calls._perform_api_call(api_call, 'get')
+    qualities = xmltodict.parse(xml_string)
+    # Minimalistic check if the XML is useful
+    if 'oml:data_qualities_list' not in qualities:
+        raise ValueError('Error in return XML, does not contain '
+                         '"oml:data_qualities_list"')
+    assert type(qualities['oml:data_qualities_list']['oml:quality']) == list
```

Comment:
Please use `isinstance()` here. Also, there's an argument to xmltodict called enforce_list which converts a part of the XML to a list so you don't need this check.

Reply:
Alright. I was following the structure for other listing functions which uses `assert`. Should it be changed for the other listing functions too for this PR?